lib/YuiRestClient.pm

```diff
@@ -54,9 +53,16 @@ sub connect_to_app {
 }
 
 sub process_start_shell {
-    assert_screen('startshell', timeout => 500);
-    type_string_slow "extend libyui-rest-api\n";
-    type_string_slow "exit\n";
+    if (get_var('S390_ZKVM')) {
+        wait_serial('ATTENTION: Starting shell', 120) || die "start shell didn't show up";
+        save_svirt_pty;
```

Comment:
Same question here, as start shell appears after we spawn the VM, so this call should be already done in bootloader, isn't it?

Reply:
very likely [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L788) and I also spot it in first screenshot in bootloader, good catch. Re-running verification: https://openqa.suse.de/tests/5107441

Reply:
It didnt' work, [ambiguous redirect error](I got https://openqa.suse.de/tests/5107441) I will keep then.

Reply:
That's strange, but doesn't harm to call it here too, we can address it later on. Cheers.